pkg/node/node.go

```diff
@@ -0,0 +1,50 @@
+/*
+© 2021 Red Hat, Inc. and others
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package node
+
+import (
+	"os"
+
+	"github.com/pkg/errors"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+)
+
+const (
+	NATDiscoveryPortLabel = "submariner.io/nat-discovery-port"
+)
+
+func GetLocalNode(cfg *rest.Config) (*v1.Node, error) {
+	nodeName, ok := os.LookupEnv("NODE_NAME")
+	if !ok {
+		return nil, errors.New("error reading the NODE_NAME from the environment")
+	}
+
+	clientset, err := kubernetes.NewForConfig(cfg)
+	if err != nil {
+		return nil, errors.Wrapf(err, "creating Kubernetes clientset")
+	}
+
+	node, err := clientset.CoreV1().Nodes().Get(nodeName, metav1.GetOptions{})
+	if err != nil {
+		return nil, errors.Wrapf(err, "unable to find own node %q", nodeName)
```

Comment:
```suggestion
		return nil, errors.Wrapf(err, "unable to find local node %q", nodeName)
```

"Own" isn't super clear to me, is this more clear?

Reply:
Done.